### DISCUSSION TITLE
速读

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2576] [插件说明]
描述
---

> 为当前页面提供速读功能，尤其适合较长页面时，可以通过次功能快速浏览当前页面的主内容。

地址
---

[https://simpread.ksria.cn/plugins/details/zsnryB7ukj](https://simpread.ksria.cn/plugins/details/zsnryB7ukj)

功能
---

1. 快捷键 `s a` 转换

2. 当进入阅读模式后自动获取当前页面的速读

选项
---

![image](https://user-images.githubusercontent.com/81074/126886541-769016c9-726e-451a-8679-c44a30927206.png)

![image](https://user-images.githubusercontent.com/81074/126886564-f6e40523-c04d-453e-bb2f-36febca75933.png)

如何使用
---

> 因为此功能使用了 [第三方 API](https://rapidapi.com/tldrthishq-tldrthishq-default/api/tldrthis/)，所以需要注册（以便获取自己的 Token）

1. 打开 https://rapidapi.com/tldrthishq-tldrthishq-default/api/tldrthis/
2. 注册（右上角 Sign Up）

   ![image](https://user-images.githubusercontent.com/81074/126886603-ebe568aa-205f-4fdb-9b29-11bbf6c1fe9d.png)

3. 输入用户名 /  邮箱地址 / 密码即可

   ![image](https://user-images.githubusercontent.com/81074/126886678-5694474c-e674-4dd0-bf58-7596a869c074.png)

4. 注册后，请在你的邮件里面找到验证码，并输入到下面界面

    ![image](https://user-images.githubusercontent.com/81074/126886761-89c0a91f-35fa-4b17-9e61-d0009e244f45.png)

5. 注册成功后，会让你填写一些必要信息：全名 /  组织名（随意填写）

  ![image](https://user-images.githubusercontent.com/81074/126886811-1ac11df9-4168-4418-af8e-03c2b2792866.png)

6.  （在注册成功的页面）直接进入 https://rapidapi.com/tldrthishq-tldrthishq-default/api/tldrthis/

7. 在下面的界面复制你的 Token

   ![image](https://user-images.githubusercontent.com/81074/126886920-c2c6d736-b9ed-458d-adf2-737563f834f6.png)

8. 将上步复制的 Token 粘贴到设置

   ![image](https://user-images.githubusercontent.com/81074/126886948-2f540b45-9f82-4f6c-b269-bcf1522bb6a3.png)

9. 完全全部设置

更新日志
---

### 1.0.1

- `Optimize` 为配合 [插件管理器](https://github.com/Kenshin/simpread/discussions/4221) 的更新方式做了数据结构的优化。

### 1.0.0

- 显示摘要
- 支持快捷键 `s a`
- 设置
    -  进入阅读模式是否自动获取

引申
---

> 安装插件或带有插件触发器的插件失效的 [解决方案](https://github.com/Kenshin/simpread/discussions/2342)。

**Comment by Renjie1997** [comment]
好像网页都说没有摘要，这正常吗

**Reply by Kenshin** [reply]
嗨

1. 这个插件效果不是很好。（跟现在比的话
2. 你可以试试这个插件 https://github.com/Kenshin/simpread/discussions/5158
